### PR TITLE
Vmo 2592 styling cleanup

### PR DESCRIPTION
**Pull request by bzabos**
This was my progress on styling and a couple bug fixes.

---

Screen Shot 2020-08-04 at 8 22 58 AM

### CODE REVIEW

File: src/store/flow/block.ts [hunk 1]
```diff
@@ -112,18 +112,27 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
...
 
-    const {destinationBlock: firstDesinationBlockId} = findBlockExitWith(first.exitId, firstBlock)
-    const {destinationBlock: secondDesinationBlockId} = findBlockExitWith(second.exitId, secondBlock)
+    const {destinationBlock: firstDestinationBlockId} = findBlockExitWith(first.exitId, firstBlock)
+    const {destinationBlock: secondDestinationBlockId} = findBlockExitWith(second.exitId, secondBlock)
+
+
+
+
+    // todo: this works only when the exit we're targetting is empty
```
Comment (bzabos) [inline src/store/flow/block.ts:121]: @safydy-r I will have to fix this in the future; I have it on my todo list. I'm going to leave these messy comments in here for now? 🤷‍♀️

**Review by bzabos — COMMENTED** [review]
@safydy-r I think I'll just merge this if I don't hear from you this evening to get things moving along into QA! :)

File: src/components/common/PlainDraggable.vue [hunk 2]
```diff
@@ -56,26 +56,38 @@
 
     mounted() {
       // todo: modify this to instantiate blank draggable onCreate, then set options when props change
+      console.debug('PlainDraggable.vue', 'mounted')
 
       // this.$nextTick(() => {
```
Comment (safydy-r) [inline src/components/common/PlainDraggable.vue:61]: Do we need to keep this comment?

File: src/store/flow/block.ts (hunk 1 above)
Comment (safydy-r) [reply src/store/flow/block.ts:121]: it's ok I think

**Review by safydy-r — APPROVED** [review]

**Pull request merged**